tracing-extensions/modules/ballerina-zipkin-extension/README.md

```diff
@@ -15,8 +17,16 @@ name="zipkin"
 [b7a.observability.tracing.zipkin]
 reporter.hostname="localhost"
 reporter.port=9411
+
+# The below properties may needed if you are need to use Zipkin V1 to send the APIs.
+#reporter.api.context="/api/v1/spans" 
+#reporter.api.version="v1"
+
+# The below properties can allow to disable the compression if needed. By default it's enabled. 
+#reporter.compression.enabled=false
+
 ```
 - Run your Ballerina service with that `ballerina.conf` file.
   - Either place `ballerina.conf` in your applications directory.
-  - Or use `-Bballerina.conf=path/to/ballerina.conf`
+  - Or use `--conf path/to/ballerina.conf`
```

Comment:
This should be `--config`
Shall we give the entire command? i.e. `ballerina run <bal-file.bal> --config path/to/ballerina.conf`

Reply:
Fixed in ee8dabf6dabb54eba8ebeafdafb072c52047bcd9.